### PR TITLE
adds cancel msg for wics & csc event

### PR DESCRIPTION
**Pull request by claricen**

### CODE REVIEW

File: content/S2016/S16-event-wics-csc.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ Slug: wics-csc-s16
 Author: Clare Ng
 Summary: We're having a bonfire with the CSC! There will be ice cream and marshmellows.
 
+**This event has been cancelled due to fire ban for KW and surrounding areas due to dry
```
Comment (evykassirer) [inline content/S2016/S16-event-wics-csc.md]: 80 character width please!

**Comment by claricen** [comment]
changed :)

File: content/S2016/S16-event-wics-csc.md [hunk 2]
```diff
@@ -4,10 +4,16 @@ Category: Events
 Tags: social
 Slug: wics-csc-s16
 Author: Clare Ng
-Summary: We're having a bonfire with the CSC! There will be ice cream and marshmellows.
+Summary: We're having a bonfire with the CSC! There will be ice cream and 
+marshmellows.
```
Comment (evykassirer) [inline content/S2016/S16-event-wics-csc.md]: not the summary though ^_^ because this is like meta data and has to be all one on line

**Comment by claricen** [comment]
noted!

**Pull request merged**